shell/components/form/SchedulingCustomization.vue

```diff
@@ -42,7 +42,7 @@ export default {
       const pdbMismatch = !!this.value?.podDisruptionBudget && (this.value?.podDisruptionBudget.maxUnavailable !== this.defaultPDB.maxUnavailable || this.value?.podDisruptionBudget.minAvailable !== this.defaultPDB.minAvailable);
       const pcMismatch = !!this.value?.priorityClass && (this.value?.priorityClass.value !== this.defaultPC.value || this.value?.priorityClass.preemptionPolicy !== this.defaultPC.preemptionPolicy);
 
-      return this.isEdit && (pdbMismatch || pcMismatch);
+      return this.feature && this.isEdit && (pdbMismatch || pcMismatch);
```

Comment:
logically this would be better as a condition on the if, rather than implying there's a settings Missmatch